ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.html

```diff
@@ -30,11 +30,20 @@
 
 <!-- Free text input -->
   <clr-control-container *ngIf="inputType == InputTypes.FREE_TEXT">
-    <input  [readOnly]="false" clrControl [ngModel]="freeText" (ngModelChange)="onFreeTextChange($event)"/>
-
-    <input *ngIf="(validation | async) as validationResult; else loading" [readOnly]="true" clrControl
-           [value]="validationResult.isValid ? validationResult.explained : 'Incorrect expression!' "/>
+    <input clrControl
+           [name]="name+'-freetext-'+uiid"
+           (ngModelChange)="onFreeTextChange($event)"
+           [ngModel]="freeText"
+           [readOnly]="false"
+           [size]="100" />
+  </clr-control-container>
 
+  <clr-control-container *ngIf="inputType == InputTypes.FREE_TEXT">
+    <input *ngIf="(validation | async) as validationResult; else loading" [readOnly]="true"
+           clrControl [name]="name+'-freetextvalidation-'+uiid"
+           [ngModel]="validationResult.isValid ? validationResult.explained : texts.FORM_VALIDATION_QUARTZ_CRON_CLR_ERR "
+           [cronQuartzExpressionValidator]="validationResult.isValid" [size]="100">
```

Comment:
Size 100 is also needed in the show mode
![image](https://user-images.githubusercontent.com/54136196/84115221-fb9f6900-aa2d-11ea-86a2-e9133849d0b4.png)

Reply:
> Size 100 is also needed in the show mode
> ![image](https://user-images.githubusercontent.com/54136196/84115221-fb9f6900-aa2d-11ea-86a2-e9133849d0b4.png)

fixed